services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -389,6 +389,12 @@ public RealmRepresentation getRealm() {
                 rep.setRegistrationEmailAsUsername(realm.isRegistrationEmailAsUsername());
             }
 
+            if (auth.realm().canViewIdentityProviders()) {
```

Comment:
Can you please remove this snippet of the code from your PR? As it looks that #31595 does not have anything to do with identity providers.